project/src/main/java/actions/TermsAndConditionsActions.java

```diff
@@ -0,0 +1,26 @@
+package actions;
+
+import ui.screens.TermsAndConditionsScreen;
+import utils.waiters.Waiter;
+
+import java.time.Duration;
+
+public abstract class TermsAndConditionsActions {
+	protected TermsAndConditionsScreen termsAndConditionsScreen = new TermsAndConditionsScreen();
+
+	public boolean isPresent() {
+		return new Waiter(Duration.ofSeconds(3)).isDisplayed(termsAndConditionsScreen.chkAccept);
+	}
+
+	public void accept() {
```

Comment:
This method first puts a check in checkbox and then clicks 'accept' button, so this name would not tell everything about what the method does. Maybe "checkAcceptAndClickAcceptButton"? (looks ugly though :( )